docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+- normalized expression matrix used for visulization
+- cell level metadata (barcodes, cell type, tissue of origin, etc.)
+- embedding (at least one required, UMAP, tSNE, spatial, PCA)
+
+Additionally, variable and feature level metadata can be useful to include but is not required for construction of an `AnnData` object.
+
+These components should be stored in the following locations in an `AnnData` object (for more information, please refer to the [AnnData documentation site](https://anndata.readthedocs.io/en/latest/)):
+
+<br/>
+
+<div align="center">
+
+| Component  | `AnnData` location            | Data Type                      | Notes                                                          |
+| ---------- |:-----------------------------:|:------------------------------:|:---------------------------------------------------------------|
+| raw count matrix | `adata.raw.X` or `adata.layers['raw']` | Numpy array or scipy sparse CSC matrix | Necessary, with some exceptions (see [exceptions](#alternative-assays))   |
+| normalized expression matrix | `adata.X`  | Numpy array or scipy sparse CSC matrix | Used for visualization in cellxgene explorer                   |
+| cell level metadata | `adata.obs` | Pandas dataframe | Categorical and continuous metadata shown in left and right cellxgene explorer sidebars respectively (can be used to color cells) |
+| variable/feature level metadata | `adata.var`   | Pandas dataframe|                                                             |
+| embedding                       | `adata.obsm`                           | Numpy array | Must start with the prefix 'X_' (i.e. adata.obsm['X_UMAP'])               |
+
+</div>
+
+<br/>
+
+<div align="center">
+  <b> Table: </b> Required data and `AnnData` object structure
+</div>
+
+<br/>
+
+<!---
+**Note:** Some assays, such as scATACseq or other epigenetic assays may not have a standardized way of representing a raw count matrix. While we still require there to be a matrix in location annotated as `raw`, some alternative options exist for these assays such as putting the un-normalized gene activity matrix in the `raw` slot. Since `adata.raw.X` can take matrices that are of a different dimensionality than `adata.X`, you could potentially put in a peak x cell matrix in this location instead. We leave this up to the author's discretion, but are happy to chat about options.
+--->
+
+**Note:** In addition to these data, other representations of the expression matrix (alternative normalizations, SCTransform, corrected counts from SCTransform or background corrected counts) can all be stored as `layers` in your `AnnData` object (as long as they maintain the same dimensionality of the main expression matrix used for visualization).
+
+**Note:** Information which pertains to the cellxgene schema will be stored in the `adata.uns` and `adata.obs` slots of the `AnnData` object and will be discussed in the [next section](#schema-definition).
+ 
+### Format conversion
```

Comment:
In general, I feel that a tutorial should **show me**. Have some canned samples that new curators can walk through step-by-step:

* I have a Seurat dataset
* I have an Anndata datset
*

Reply:
I agree, for future iterations